buildpipeline/osx.groovy

```diff
@@ -35,6 +35,36 @@ simpleNode('OSX10.12','latest') {
         if (params.TestOuter) {
             additionalArgs = '-Outerloop'
         }
-        sh "HOME=\$WORKSPACE/tempHome ./build-tests.sh -buildArch=x64 -${params.CGroup} ${additionalArgs} -- /p:ArchiveTests=true /p:EnableDumpling=true"
+        sh "HOME=\$WORKSPACE/tempHome ./build-tests.sh -buildArch=x64 -${params.CGroup} -SkipTests ${additionalArgs} -- /p:ArchiveTests=true /p:EnableDumpling=true"
     }
+    stage ('Submit To Helix For Testing') {
+        // Bind the credentials
+        withCredentials([string(credentialsId: 'CloudDropAccessToken', variable: 'CloudDropAccessToken'),
+                         string(credentialsId: 'OutputCloudResultsAccessToken', variable: 'OutputCloudResultsAccessToken')]) {
+            // Ask the CI SDK for a Helix source that makes sense.  This ensures that this pipeline works for both PR and non-PR cases
+            def helixSource = getHelixSource()
+            // Ask the CI SDK for a Build that makes sense.  We currently use the hash for the build
+            def helixBuild = getCommit()
+            // Get the user that should be associated with the submission
+            def helixCreator = getUser()
+            // Target queues
+            def targetHelixQueues = ['OSX.1012.Amd64.Open',
+                                     'OSX.1013.Amd64.Open',]
+
+            sh "HOME=\$WORKSPACE/tempHome ./Tools/msbuild.sh src/upload-tests.proj /p:ArchGroup=x64 /p:ConfigurationGroup=${params.CGroup} /p:TestProduct=corefx /p:TimeoutInSeconds=1200 /p:TargetOS=OSX /p:HelixJobType=test/functional/cli/ /p:HelixSource=${helixSource} /p:BuildMoniker=${helixBuild} /p:HelixCreator=${helixCreator} /p:CloudDropAccountName=dotnetbuilddrops /p:CloudResultsAccountName=dotnetjobresults /p:CloudDropAccessToken=\$CloudDropAccessToken /p:CloudResultsAccessToken=\$OutputCloudResultsAccessToken /p:HelixApiEndpoint=https://helix.dot.net/api/2017-04-14/jobs /p:TargetQueues=${targetHelixQueues.join('+')} /p:HelixLogFolder=${WORKSPACE}/${logFolder}/ /p:HelixCorrelationInfoFileName=SubmittedHelixRuns.txt"
```

Comment:
At some point we really ought to be parameterizing out the storage accounts we use.